applications/GeoMechanicsApplication/tests/cpp_tests/test_transient_thermal_element.cpp

```diff
@@ -87,44 +89,39 @@ void ValidateThermalElement(ModelPart& rModelPart)
     }
 }
 
-void GenerateTransientThermalElementWithZeroDomainSize(ModelPart& rModelPart)
+Element::Pointer GenerateTransientThermalElementWithZeroDomainSize()
 {
-    // Geometry creation
-    rModelPart.CreateNewNode(1, 0.0, 0.0, 0.0);
-    rModelPart.CreateNewNode(2, 0.0, 0.0, 0.0);
-    rModelPart.CreateNewNode(3, 0.0, 0.0, 0.0);
-    std::vector<ModelPart::IndexType> node_ids{1, 2, 3};
-    rModelPart.CreateNewElement("GeoTransientThermalElement2D3N", 1, node_ids,
-                                rModelPart.CreateNewProperties(0));
+    PointerVector<Node> nodes;
+    nodes.push_back(make_intrusive<Node>(1, 0.0, 0.0, 0.0));
+    nodes.push_back(make_intrusive<Node>(2, 0.0, 0.0, 0.0));
+    nodes.push_back(make_intrusive<Node>(3, 0.0, 0.0, 0.0));
+    auto transient_thermal_element = make_intrusive<TransientThermalElement<2, 3>>(
+        1, Kratos::make_shared<Triangle2D3<Node>>(nodes));
+
+    return transient_thermal_element;
```

Comment:
Just for my understanding: the reason you don't add the element to the model part here is because it would throw an exception, since the `Check` operation fails...?

Reply:
I don't because we don't need a model part here, so I just create the element without a model part

Reply:
Ah yes, now I see. My bad.